client/src/main/java/io/pravega/client/netty/impl/ClientConnectionImpl.java

```diff
@@ -52,9 +51,7 @@ public void send(WireCommand cmd) throws ConnectionFailedException {
     @Override
     public void send(Append append) throws ConnectionFailedException {
         checkClientConnectionClosed();
-        final AppendBatchSizeTracker batchSizeTracker = nettyHandler.getAppendBatchSizeTracker(append.getFlowId());
```

Comment:
This seems to be removing the `AppendBatchSizeTracker` altogether, is my observation and if so why?

Reply:
no.. its moved to CommandEncoder